src/Tizen.Wearable.CircularUI.Chart.Forms/PieDataItem.cs

```diff
@@ -0,0 +1,55 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Xamarin.Forms;
+
+namespace Tizen.Wearable.CircularUI.Chart.Forms
+{
+    /// <summary>
+    /// PieDataItem is an inheritance type of DataItem for the PieChartView..
+    /// </summary>
+    public class PieDataItem : IDataItem
+    {
+        public PieDataItem()
+        {
+        }
+
+        public PieDataItem(TextItem label, double value, Color color)
```

Comment:
Arguments are not used

Reply:
removed PieDataItem. no more used this class